data-structures/ds-pque/java/PriorityQueue.java

```diff
@@ -0,0 +1,65 @@
+public class PriorityQueue {
+    static class Node {
+        int data;
+        int priority;
+        Node next;
+
+    }
+
+    static Node node = new Node();
+
+    static Node newNode(int d, int p) {
+        Node temp = new Node();
+        temp.data = d;
+        temp.priority = p;
+        temp.next = null;
+
+        return temp;
+    }
+
+    static int peek(Node head) {
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6336404)